src/components/commons/error/PageError.tsx

```diff
@@ -1,38 +1,57 @@
 import { Head } from "@components/commons/Head";
 import { Container } from "@components/commons/Container";
-import { BiErrorCircle } from "react-icons/bi";
-import Link from "next/link";
+import { Button } from "@components/commons/Buttons";
+import { SectionSubTitle } from "@components/commons/SectionSubTitle";
+import classNames from "classnames";
 
 export function PageError(props: {
   title: string;
+  subtitle: string;
   errorDesc: string;
   text: string;
 }): JSX.Element {
+  const { title, subtitle, errorDesc, text } = props;
   return (
     <>
       <Head>
-        <title>{props.title}</title>
+        <title>{title}</title>
```

Comment:
for consistency, can we move the title to `<Head>`'s prop instead of children?

Reply:
ok!